_assets/stylesheets/_base.scss

```diff
@@ -19,3 +20,4 @@ $fa-font-path: "/assets/fonts";
 @import "links";
 @import "fonts";
 @import "typography";
+@import "card";
```

Comment:
This should be called `card-clear`, moved to the `components` folder, and included in `public.scss` and not `base.scss`.